packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -8,7 +8,10 @@ input DiagramEventInput {
   diagramId: ID!
 }
 
-union DiagramEventPayload = ErrorPayload | SubscribersUpdatedEventPayload | DiagramRefreshedEventPayload
+union DiagramEventPayload =
```

Comment:
You seems to be using a tool which changes the formatting of the graphqls files, please be careful with that.